napari/_qt/containers/_layer_delegate.py

```diff
@@ -153,6 +157,7 @@ def editorEvent(
 
         This can be used to customize how the delegate handles mouse/key events
         """
+        global layer_states
```

Comment:
Where is this variable defined?

Reply:
This is a commit error, a leftover from stuff that didnt work.